index.js

```diff
@@ -161,27 +230,28 @@ function runCumulusTask(taskFunction, cumulusMessage, context, callback, schemas
   }
   else {
     const promisedRemoteEvent = loadAndUpdateRemoteEvent(cumulusMessage, context, schemas);
-    const promisedNestedEvent = promisedRemoteEvent
-      .then((event) => loadNestedEvent(event, context, schemas));
+    const promisedNestedEvent = promisedRemoteEvent.then((event) => {
+      safeSetEnv('GRANULES', JSON.stringify(getMessageGranules(cumulusMessage)));
```

Comment:
this is probably wrong.  we need the remote event here not the input cumulusMessage, (that's why we need tests. duh)